### DISCUSSION TITLE
Welcome to Locust Discussions forum!

### DISCUSSION THREAD

**Discussion by cyberw** [locustio/locust#2719] [Announcements]
We have a forum now!

This is meant to augment the discussions on [StackOverflow](https://stackoverflow.com/questions/tagged/locust) and [Locust's slack](https://locustio.slack.com/) [(Slack sign up)](https://communityinviter.com/apps/locustio/locust)

Feel free to ask questions, share solutions and talk about anything Locust-related :)

**Comment by miliev1012** [comment]
Hey Lars, 

Thanks for all the work that you are doing on locust, so far was only user, hopefully at some point I can contribute. 
Now to the point, I'm trying to replicate a behavior of a massive spike of load (let say 200K events/s) via a message provider that should last totally 2-3-4 seconds and then go back to lower/normal levels(2-10K). The problem is when I start that test there is a delay(due to initial creation of Users) and then I may eventually reach those levels, but not start directly with that load.
I start wandering is there a way to actually create my test to wait some period(or when ALL users are available) and then blast the load ? 
I'm using k8s + master/slave, so maybe going with A LOT of workers is also an option, but I prefer to have a way to provide stable RPS for that initial blast of load. Also those numbers are current and hopefully(for the business) will need to test some times more than that :).

Cheers,
Manoel.

**Reply by cyberw** [reply]
Hi! Great question. I'll write an answer for you, but could you maybe make this a post in the Q&A section instead of a comment on this post (that makes it easier for other people to find, if they have the same question)
